src/api/routers/participantUsersRouter.ts

```diff
@@ -0,0 +1,23 @@
+import express from 'express';
+
+import { UserController } from '../controllers/userController';
+import { enrichWithUserFromParams } from '../middleware/usersMiddleware';
+import { LoggerService } from '../services/loggerService';
+import { UserService } from '../services/userService';
+
+const createParticipantUsersRouter = () => {
+  const participantUsersRouter = express.Router({ mergeParams: true });
```

Comment:
`mergeParams` allows this router to access the params from the parent router, in this case, `participantId`.